docs/src/main/asciidoc/security-oidc-code-flow-authentication.adoc

```diff
@@ -52,11 +52,12 @@ For information about how to support multiple tenants, see xref:security-openid-
 
 === Configuring access to the OIDC provider endpoint
 
-The OIDC `web-app` application requires URLs of the OIDC provider's authorization, token, `JsonWebKey` (JWK) set, and possibly the `UserInfo`, introspection, and end session (RP-initiated logout) endpoints.
+The OIDC `web-app` application requires URLs of the OIDC provider's authorization, token, `JsonWebKey` (JWK) set, and possibly the `UserInfo`, introspection and end-session (RP-initiated logout) endpoints.
```

Comment:
I will prepare a slight rework for this section, but it will be done in the following PR.

Reply:
Hi @MichalMaler, did you have additional feedback for this section to add here?